src/main/scala/org/broadinstitute/dsde/rawls/model/MethodRepoModel.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.rawls.model
+
+import com.wordnik.swagger.annotations.{ApiModelProperty, ApiModel}
+import org.joda.time.DateTime
+
+import scala.annotation.meta.field
+
+object AgoraEntityType extends Enumeration {
+  type EntityType = Value
+  val Task = Value("Task")
+  val Workflow = Value("Workflow")
+  val Configuration = Value("Configuration")
+}
+
+@ApiModel(value = "AgoraEntity")
+case class AgoraEntity(
```

Comment:
we should call things either Agora or MethodRepository, but not have both. Agora is an implementation of a method repo so I am inclined to go with the latter.